src/hooks/useClientSideV3Trade.ts

```diff
@@ -34,15 +34,22 @@ export function useClientSideV3Trade<TTradeType extends TradeType>(
       : [otherCurrency, amountSpecified?.currency]
   const { routes, loading: routesLoading } = useAllV3Routes(currencyIn, currencyOut)
 
-  const quoter = useV3Quoter()
   const { chainId } = useWeb3React()
+  const useQuoterV2 = useMemo(
```

Comment:
Add a comment here

    // Chains deployed using the deploy-v3 script only deploy QuoterV2.